### PR TITLE
Edits to DOCS-1076 - Documenting PGD 5.6.1 #6210

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5.6/ddl/ddl-command-handling.mdx [hunk 1]
```diff
@@ -253,11 +253,11 @@ Some variants of `ALTER TABLE` currently aren't allowed on a PGD node:
 -   `ADD COLUMN ... DEFAULT (non-immutable expression)` &mdash; This is not allowed because
     it currently results in different data on different nodes. See
     [Adding a column](ddl-workarounds/#adding-a-column) for a suggested workaround.
-    This can be overriden using `bdr.permit_unsafe_commands` if user is sure the command is
+    This can be overridden using `bdr.permit_unsafe_commands` if user is sure the command is
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5.6/ddl/ddl-command-handling.mdx:256]: Is there some reason this is in third person and passive voice? Suggest this revision:

```suggestion
    You can override this behavior using `bdr.permit_unsafe_commands` if you're sure the command is
```

File: product_docs/docs/pgd/5.6/ddl/ddl-command-handling.mdx [hunk 2]
```diff
@@ -253,11 +253,11 @@ Some variants of `ALTER TABLE` currently aren't allowed on a PGD node:
...
     safe.
 -   `ALTER COLUMN ... SET STORAGE external` &mdash; Is rejected if the column is
     one of the columns of the replica identity for the table.
-    This can be overriden using `bdr.permit_unsafe_commands` if user is sure the command is
+    This can be overridden using `bdr.permit_unsafe_commands` if user is sure the command is
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5.6/ddl/ddl-command-handling.mdx:260]: Same query. Suggest:
```suggestion
    You can override this behavior using `bdr.permit_unsafe_commands` if you're sure the command is
```

File: product_docs/docs/pgd/5.6/index.mdx [hunk 3]
```diff
@@ -74,7 +74,7 @@ the peer nodes only after the local commit. You can configure additional levels
 
 ## Compatibility
 
-EDB Postgres Distributed 5 is compatible with PostgreSQL, EDB Postgres Extended, and EDB Postgres Advanced versions 12-17. See the [compatibility](compatibility) section for more details, including compatibility with different operating systems and architectures.
+EDB Postgres Distributed 5 is compatible with PostgreSQL, EDB Postgres Extended, and EDB Postgres Advanced versions 12-17. See the [Compatibility](compatibility) for more details, including compatibility with different operating systems and architectures.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5.6/index.mdx:77]:
```suggestion
EDB Postgres Distributed 5 is compatible with PostgreSQL, EDB Postgres Extended, and EDB Postgres Advanced versions 12-17. See [Compatibility](compatibility) for more details, including information about compatibility with different operating systems and architectures.
```

File: product_docs/docs/pgd/5.6/conflict-management/conflicts/02_types_of_conflict.mdx [hunk 4]
```diff
@@ -33,13 +33,17 @@ You can effectively eliminate this type of conflict by using [global sequences](
...
+In either of the following cases, applying the replication change produces a `multiple_unique_conflicts` conflict. Both of these cases result in a conflict against more than one other row.
 
-In case of such a conflict, you must remove some rows for replication to continue. 
-Depending on the resolver setting for `multiple_unique_conflicts`, the apply process either exits with error, skips the incoming row, or deletes some of the rows. 
+- If a new row violates more than one `UNIQUE` constraint and that results in a conflict against more than one other row. 
+
+- If a new row violates more than one `EXCLUDE` constraint or a single `EXCLUDE` constraint.
+
+In case of such a conflict, for replication to continue, you must remove some rows. 
+Depending on the resolver setting for `multiple_unique_conflicts`, the apply process either exits with an error, skips the incoming row, or deletes some of the rows.
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5.6/conflict-management/conflicts/02_types_of_conflict.mdx:46]: I'm happy. @jpe442 want to look?

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**